### ISSUE TITLE
ETC5543 report issue

### ISSUE THREAD

**Issue by Yiwen-Zhang-259** [Yiwen-Zhang-259/ETC5543-cran-analysis#19]
Hi Emi and Hien,

I have almost finished my first draft. Could you please take a look at my problems and give me some suggestions？ I would appreciate it . : D

**Comment by emitanaka** [comment]
Hien probably needs to be tagged to see this @hiendn

**Comment by hiendn** [comment]
Thanks Emi, Yiwen, I'll have a look at it over the weekend.

**Comment by Yiwen-Zhang-259** [comment]
@hiendn  That would be nice, thanks Hien!

**Comment by emitanaka** [comment]
@Yiwen-Zhang-259 isn't this due today?

**Comment by Yiwen-Zhang-259** [comment]
@emitanaka , yes, but as I have something wrong with building the github book and hope to get advice from Hien as well, so I apply for extension until next Monday 😄 Meanwhile, I can't build github book with is error :
![image](https://user-images.githubusercontent.com/62992147/121646422-e1c4b600-cac7-11eb-8696-603bc2defa63.png)
And these "undefined" refs are indeed in my book.bib file  for when just knitting y report locally, the refs would be ok.  Maybe it is due to output.yaml ? Thanks Emi !

**Comment by Yiwen-Zhang-259** [comment]
@emitanaka  Hi Emi, I have fixed the bib file issue and build the Githubbook successfully ! Thanks for your generous help and patience during this research project! I'm sorry I didn't perform very outstanding. It's a great honor  and  big pleasure for me to have this opportunity to work under your supervision ! And also thanks a lot for Hien @hiendn , for giving me so many nice ideas and help!  (btw in case of you couldn't see the last post I comment : As I am afraid I couldn't receive the link of ref from you (as it  was quite late in Melbourne that time and weekend is coming ) and build Githubbook within  today, and also I would like to hope receive advice from Hien, I applied for an extension from Rob until Monday 2hours ago, sorry forget to tell you about that! 😊）

**Comment by hiendn** [comment]
@Yiwen-Zhang-259 its been a pleasure. Did you still want me to read over your writing? That’s what I’m mainly good for. Emi is the expert programmer.

**Comment by Yiwen-Zhang-259** [comment]
@hiendn  Yes, please! I would very appreciate it if you could give me some advice on the writing! 😃

**Comment by hiendn** [comment]
Hi @Yiwen-Zhang-259, I've made some comments to the text from Chapters 1--4. I tried to comment on the text from Chapters 5--7, but for some reason my Adobe reader keeps crashing. So hopefully these comments can get you started on your corrections!

Just one large comment regarding Chapters 5--7; all of the text appears to be center aligned. You will want to make the text justified to be more like the rest of the document up to that point.

Here is a link to my commented document for you (you will need Adobe reader to read the comments): https://www.dropbox.com/s/tifiwf5onwfaebm/ETC5543-Yiwen-Zhang_HN_edit.pdf?dl=0

**Comment by Yiwen-Zhang-259** [comment]
@hiendn , Thank you so much , Hien! I will take a look at what you commented very carefully and started my correction!
